bindings/python/src/pgvecto_rs/sdk/record.py

```diff
@@ -0,0 +1,45 @@
+from typing import List, Optional, Type, Union
+from uuid import UUID, uuid4
+
+from numpy import array, float32, ndarray
+from sqlalchemy.orm import DeclarativeBase, Mapped
+
+
+class RecordORM(DeclarativeBase):
+    __tablename__: str
+    id: Mapped[UUID]
+    document: Mapped[dict]
+    embedding: Mapped[ndarray]
+
+
+RecordORMType = Type[RecordORM]
+
+
+class Record:
+    id: UUID
+    document: dict
+    embedding: ndarray
+
+    def __init__(self, id: UUID, document: dict, embedding: ndarray):
+        self.id = id
+        self.document = document
+        self.embedding = embedding
+
+    def __repr__(self) -> str:
+        return f"""============= Record =============
+[id]       : {self.id}
+[document]     : {self.document}
+[embedding]: {self.embedding}
+========== End of Record ========="""
+
+    @classmethod
+    def from_orm(cls, orm: RecordORM):
+        return cls(orm.id, orm.document, orm.embedding)
+
+    @classmethod
+    def from_text(
+        cls, document: Optional[dict], embedding: Union[ndarray, List[float]]
```

Comment:
Using the dict parameter is a bad idea. It's only used when you have to. BTW, `from_text` but has no `text` required is also a bad interface.

Filtering is an add-on feature, not an everyone-must-use feature. Users have to create a dict even when they don't have any metadata for filtering.